src/Core/src/Handlers/Picker/PickerHandler.Android.cs

```diff
@@ -29,6 +30,7 @@ protected override void DisconnectHandler(MauiPicker platformView)
 		{
 			platformView.FocusChange -= OnFocusChange;
 			platformView.Click -= OnClick;
+			platformView.LayoutChange -= PlatformViewLayoutChange;
```

Comment:
This will fire for everything as noted by the comment https://github.com/dotnet/maui/issues/21717#issuecomment-2333824320

Does this need to be all the time or just the first time? What happens if you changed the grid row height from say 200 to 400, is the alignment wrong again?